README.md

```diff
@@ -0,0 +1,69 @@
+css-literal-loader
+====
+
+A webpack loader for extracting and processing css defined in other files.
+
+"Inline css" that just works with css modules, extract-text-webpack-plugin, postCSS, LESS, SASS, or any other css preprocessor or loader.
+
+```js
+import React from 'react';
+
+const styles = css`
+  .button {
+    color: black;
+    border: 1px solid black;
+    background-color: white;
+  }
+`;
+
+export default function Button(props) {
+  return <button className={styles.button} {...props} />
```

Comment:
maybe just use `children` in this example? `props.className` could potentially clobber our `className`